lib/ex_force/response.ex

```diff
@@ -0,0 +1,26 @@
+defmodule ExForce.Response do
+  @moduledoc """
+  Represents an ExForce API response.
+  """
+
+  @type param() :: binary() | [{binary() | atom(), param()}]
+  @type t :: %__MODULE__{
+          body: any(),
+          headers: [{binary(), binary()}],
+          method: :head | :get | :delete | :trace | :options | :post | :put | :patch,
+          opts: [any()],
+          query: [{binary() | atom(), param()}],
```

Comment:
Same here for param in query

Reply:
Fixed with 64ad54e108e733b617ceba9701347e181e2eacce